pkg/frontend/adminactions/azureactions.go

```diff
@@ -5,6 +5,7 @@ package adminactions
 
```

Comment:
why does admin actions need applens?

Reply:
I added it to Azure Actions based on nothing more than the fact that this would be connecting to an Azure endpoint. Azure Actions does fall under Admin Actions, but the endpoints I added are not admin endpoints.

A separate actions class could be made if desired/need be.

Reply:
i see. i think that's fine by me